plugins/Colmena/ErrorsManager/src/Model/Table/CompilationsTable.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Colmena\ErrorsManager\Model\Table;
+
+use App\Model\Table\AppTable;
+use Cake\Validation\Validator;
+use App\Encryption\EncryptTrait;
+
+/**
+ * Student Model.
+ *
+ */
+class CompilationsTable extends AppTable
+{
+
+    /**
+     * Initialize method.
+     *
+     * @param array $config the configuration for the Table
+     */
+    public function initialize(array $config): void
+    {
+        parent::initialize($config);
+
+        $this->setTable('em_compilations');
+        $this->setDisplayField('name');
+        $this->setPrimaryKey('id');
+
+        // $this->hasOne('Family', [
```

Comment:
why is this relationship commented?

Reply:
You are right, that shouldn't exist either. That relationship is residual.